signup/impl/src/java/org/sakaiproject/signup/logic/SakaiFacadeImpl.java

```diff
@@ -1464,4 +1464,17 @@ private void disableSecurityAdvisor(SecurityAdvisor securityAdvisor){
 		securityService.popAdvisor(securityAdvisor);
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public User getUserByDisplayId(String displayId) {
+		User user = null;
+		try {
+			user = userDirectoryService.getUserByAid(displayId);
```

Comment:
I don't understand this ..... getUserByDisplayId takes the display id and then calls getUserByAid ?